include/alpaka/acc/AccGenericSycl.hpp

```diff
@@ -51,7 +51,7 @@ namespace alpaka::experimental
     //!
     //! This accelerator allows parallel kernel execution on SYCL devices.
     template<typename TDim, typename TIdx>
-    class AccGenericSycl
+    class AccGenericSycl final
```

Comment:
This must not be `final` because it is the abstract base class for the concrete SYCL implementations.